### ISSUE TITLE
Rule Idea: item description should not be the same like item name

### ISSUE THREAD

**Issue by wernerkrauss** [sindresorhus/awesome-lint#125]
This should avoid something like:

[Some Item Name](www.example.com) - Some Item Name.

**Comment by sindresorhus** [comment]
Good idea. Could we take it as far as saying that the description should not even contain the title?

**Comment by wernerkrauss** [comment]
> Could we take it as far as saying that the description should not even contain the title?

Phew, could there be valid cases where the title can be in the description? At least it should not start with the title. So this is invalid:

[Some Item Name](www.example.com) - Some Item Name is totally awesome.

In a PR I received this today:

- [SSPy](https://github.com/Firesphere/silverstripe-sspy) - Python version of SSPak, that can handle larger assets. SSPak supports up to 2GB, SSPy has no limit. It is however, less easy to install and set-up.

It has "SSPy" in the description and I tend to accept it. OTOH one could rewrite it like "This module has no limit" to avoid the title in the description.

**Comment by sindresorhus** [comment]
Ok, let's do "description cannot **start with**".
